packages/eslint-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/eslint-plugin",
-  "version": "1.18.1",
+  "version": "1.18.2",
```

Comment:
@harshil1793 Were you able to create a new release for this package yet and update vets-website?